src/main/java/fr/siamois/infrastructure/database/initializer/seeder/customform/CustomFormPanelDTO.java

```diff
@@ -0,0 +1,14 @@
+package fr.siamois.infrastructure.database.initializer.seeder.customform;
+
+
+
+import java.io.Serializable;
+import java.util.List;
+
+
+public record CustomFormPanelDTO (
+    String className,
+    String name,
+    List<CustomRowDTO> rows,
+    Boolean isSystemPanel // define by system or user?
```

Comment:
```suggestion
    bool isSystemPanel // define by system or user?
```
Est-ce que utiliser un Wrapper est vraiment utile ? Si ce n'est pas le cas, c'est plus lisible/facile d'utiliser la primitive `boolean`